config/locales/account/confirmations.en.yml

```diff
@@ -28,18 +28,10 @@ en:
             <p class="govuk-body">You need to confirm your new email address.</p>
             <p class="govuk-body">We’ve sent an email to <strong>%{email}</strong>. Click on the confirmation link in the email to finish updating the email address for your GOV.UK account.</p>
       finish_creating:
-        details:
-          brexit: |
-            <p class="govuk-body">You also need to confirm your email address.</p>
-            <p class="govuk-body">We’ve sent an email to <strong>%{email}</strong>. Click on the confirmation link in the email to:</p>
-            <ul class="govuk-list govuk-list--bullet">
-              <li>finish creating your account</li>
-              <li>start receiving email updates about Brexit</li>
-            </ul>
-          dashboard: |
-            <p class="govuk-body">You need to confirm your email address.</p>
-            <p class="govuk-body">We’ve sent an email to <strong>%{email}</strong>. Click on the confirmation link in the email to finish creating your account.</p>
-        must_continue: You must continue to your Brexit checker results to finish saving them to your account.
+        address_confirmation: |
+          <p class="govuk-body">You'll also need to confirm your email address. Click on the confirmation link we've emailed to <strong>%{email}</strong>.</p>
```

Comment:
Sorry to be pedantic but can we use the correct character for the apostrophes in `You'll`, `you've` etc? 
I thought the convention was to use the curly one like `we’ve` instead of the straight one like `we've`.